static_frame/core/display.py

```diff
@@ -22,6 +23,9 @@
 from static_frame.core.display_config import _DISPLAY_FORMAT_TERMINAL
 from static_frame.core.style_config import StyleConfig
 
+if tp.TYPE_CHECKING:
+    from static_frame.core.index_base import IndexBase # pylint: disable=unused-import
```

Comment:
Please add coverage exclusion too. 
```suggestion
    from static_frame.core.index_base import IndexBase # pylint: disable=unused-import #pragma: no cover
```

Reply:
This change came in from default :+1: